packages/instrumenter/src/mutant-placers/expression-mutant-placer.ts

```diff
@@ -60,7 +60,7 @@ function isMemberOrCallExpression(path: NodePath) {
 }
 
 function isMemberExpression(path: NodePath): path is NodePath<types.MemberExpression | types.OptionalMemberExpression> {
-  return path.isMemberExpression() || path.isOptionalMemberExpression();
+  return path.isMemberExpression() || path.isOptionalMemberExpression() || path.isTSNonNullExpression();
```

Comment:
This breaks typesafety, as the return type specifies it to be a member expression. Also, the method name is no longer correct 🤷‍♂️

Reply:
Good point! I forgot to change that

Reply:
The NonNullExpression seems like a MemberExpression to me, so I think the method name does seem valid.